spec/requests/articles/articles_destroy_spec.rb

```diff
@@ -14,11 +14,11 @@
     expect(destroyed_article).to be_nil
   end
 
-  it "schedules a RemoveAllJob if there are comments" do
+  it "schedules a RemoveAllWorker if there are comments" do
     create(:comment, commentable: article, user: user)
-    expect do
+    sidekiq_assert_enqueued_jobs(2, only: Notifications::RemoveAllWorker) do
```

Comment:
here we went from an assertion that checks for 1 job, to one that checks for 2 jobs, what changed?